.gitlab-ci.yml

```diff
@@ -17,7 +17,7 @@ variables:
   # Run on the Internal cluster only on commit.
   # We run the tests via the integration test framework for Non-Prod/Prod.
   DEPLOY_INTERNAL: "true"
-  DEPLOY_NONPROD: "false"
+  DEPLOY_DEV: "false"  # runs on Non-Prod
```

Comment:
I have mixed feeling towards this renaming. I'd personally prefer:
- `DEPLOY_INTERNAL` -> `DEPLOY_DEV `
- `DEPLOY_NONPROD ` -> `DEPLOY_STAGE`

Considering existing naming, we should either use cluster name across the board, being `sandbox`, `dev`, `stage`, `prod`, or we use environment name being `sandbox`, `internal`, `nonprod`, `prod`.

Thinking from actual development cycle, we do try things out in sandbox and develop against internal cluster. We as AIP engineers do not "dev" in NonProd cluster since it should be a production grade environment.

Reply:
Me too, this is following aip-k8s-manifests pattern set in:
* https://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-infrastructure/aip-k8s-manifests/-/blob/be735e33c9f390af2f17538fd04818e0a00b46ec/pytest/tests/projects/__init__.py#L23

Reply:
Thank you! fixed along with https://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-infrastructure/aip-k8s-manifests/-/merge_requests/483!